cli/package.json

```diff
@@ -61,11 +61,13 @@
     "@types/gradient-string": "^1.1.2",
     "@types/inquirer": "^9.0.3",
     "@types/node": "^18.15.5",
+    "eslint-config-prettier": "^8.8.0",
     "next": "^13.2.4",
     "next-auth": "^4.20.1",
-    "prettier": "^2.8.6",
+    "prettier": "^2.8.7",
+    "prettier-plugin-prisma": "^4.12.0",
```

Comment:
why would we need a prettier plugin here? the prisma cli has a `format` command and their VSCode extension is probably a better recommendation than a 3rd party prettier plugin?